pkgs/development/ocaml-modules/mdx/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, buildDunePackage, astring, cmdliner, cppo, fmt, logs, lwt4, ocaml-migrate-parsetree, pandoc, re }:
+
+buildDunePackage rec {
+  pname = "mdx";
+  version = "1.4.0";
+
+  minimumOCamlVersion = "4.02.3";
```

Comment:
Something is wrong with OCaml 4.04:

> /nix/store/cnd05hmvxdvrj5fcl54868nn24b2q815-ocaml4.04.2-mdx-1.4.0/bin/ocaml-mdx-test not found or is not a bytecode executable file

Maybe this lower-bound should be raised.

It works well with 4.03 though…

Reply:
> Maybe this lower-bound should be raised.

Done